### PR TITLE
Feature/improve connector via url

### PR DESCRIPTION
**Pull request by chiro-hiro**
## Type

- [x] Feature
- [ ] Bug fix
- [ ] CI/CD
- [ ] Documentation
- [ ] Other

## Description

Brief description of the changes made.

### CODE REVIEW

File: packages/zkdb/src/sdk/global/zkdatabase-client.ts [hunk 1]
```diff
@@ -1,29 +1,82 @@
...
+   * ```
+   * Connect from browser using Auro Wallet
+   * ```ts
+   * const client = await ZKDatabaseClient.connect('zzkdb+https://username@test-serverless.zkdatabase.org/graphql?db=my-db');
+   * ```
+   * @param url
+   * @returns
+   */
+  public static async connec(url: string): Promise<ZKDatabaseClient> {
+    const urlInstance = new URL(url);
```
Comment (SangTran-127) [inline packages/zkdb/src/sdk/global/zkdatabase-client.ts:40]: spell check

File: packages/zkdb/src/sdk/global/zkdatabase-client.ts [hunk 2]
```diff
@@ -1,29 +1,82 @@
 import { ApiClient, IApiClient } from '@zkdb/api';
-import { Authenticator, ISecureStorage } from '../authentication';
+import { Authenticator } from '../authentication';
 import { ZKDatabase, GlobalContext } from '../interfaces';
 import { ZKDatabaseImpl, GlobalContextImpl } from '../impl';
 
-import { Signer } from '../signer';
+import { AuroWalletSigner, NodeSigner, Signer } from '../signer';
+import { PrivateKey } from 'o1js';
```
Comment (SangTran-127) [inline packages/zkdb/src/sdk/global/zkdatabase-client.ts:7]: import external package first

File: packages/zkdb/src/sdk/storage/memory.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+export class InMemoryStorage implements Storage {
+  [name: string]: any;
```
Comment (SangTran-127) [inline packages/zkdb/src/sdk/storage/memory.ts:2]: what [name: string] do?

Comment (chiro-hiro) [reply packages/zkdb/src/sdk/storage/memory.ts:2]: It's key value

**Review by SangTran-127 — APPROVED** [review]

**Review by SangTran-127 — APPROVED** [review]

**Pull request merged**